lib/spack/spack/schema/config.py

```diff
@@ -87,6 +87,7 @@
                 ],
             },
             'allow_sgid': {'type': 'boolean'},
+            'binary_distribution_cache_root': {'type': 'string'},
```

Comment:
Sorry one last thing: I think this needs to be `binary_index_root` (I requested a name change in `binary_distribution.py` which was made there for `config:binary_index_root`).

Reply:
Dang it!  Should be just a sec.

Reply:
I'm sorry I missed that, but thank you for catching it.  I just pushed the fix.